client/src/components/Ratings/sub-components/ReviewList.jsx

```diff
@@ -139,12 +146,12 @@ export default function ReviewList({
         }
       </div>
       <div>
-        <ReviewSearchBar
+        {/* <ReviewSearchBar
           reviews={reviews}
           searchTerm={searchTerm}
           setSearchTerm={setSearchTerm}
           listIndex={listIndex}
-        />
+        /> */}
```

Comment:
Do you still need this?

Reply:
Nope! Was an artifact from shifting positioning. deleted.